src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -447,7 +448,7 @@ export function ChangeDataView({
         buffer={8}
       >
         <div css={styles.popoverContent}>
-          <EuiContextMenuPanel size="s" items={getPanelItems()} />
+          <EuiContextMenuPanel size="s" items={getPanelItems()} key="dataview-picker-menu" />
```

Comment:
Should not be needed the key here as it is not part of an array

Reply:
Correct! https://github.com/elastic/kibana/pull/148296/commits/c688932a39d4fffe7321164a4094122aab5f7514